2017/AUG/AUG17/PALINGAM/PALINGAM.py

```diff
@@ -0,0 +1,37 @@
+#Code Copyright: Amit Sarkar, Computer Science Engineering 2nd Year, BIT Mesra
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpq19l0tf7/2017/AUG/AUG17/PALINGAM/PALINGAM.py
+++ b/tmp/tmpq19l0tf7/2017/AUG/AUG17/PALINGAM/PALINGAM.py
@@ -1,4 +1,4 @@
-#Code Copyright: Amit Sarkar, Computer Science Engineering 2nd Year, BIT Mesra
+# Code Copyright: Amit Sarkar, Computer Science Engineering 2nd Year, BIT Mesra
 t = input()
 
 while ( t > 0 ) :
```